### DISCUSSION TITLE
Can a custom publication date be set with year only?

### DISCUSSION THREAD

**Discussion by jeffpooley** [knowledgefutures/pubpub#1839] [General]
At mediastudies.press we sometimes publish public domain Pubs that don’t have specific day/months—only a year. Here’s an [example](https://www.mediastudies.press/pub/ee890kcf/draft?access=3qabjxtu). Our workaround is to use something like January 1, [Year], but just checking if there’s a way to designate year only?

**Comment by gabestein** [comment]
Hey Jeff — thanks! Fun one we hadn't considered before. Time is weird...

Unfortunately, the answer is no, not right now at the Pub level (at the collection level if it's a book, yes, mainly because bibtex and Crossref support a year-only field).

We're gonna have to talk internally about what it would mean to support year-only publication dates. Behind the scenes, it's tricky because that database field is a full timestamp, so there's technically no way to enter in just a year, I don't think (we need better datatypes!). But I wonder if we could allow the user to specify a date format or something like that to at least make it work on the display/citation level.

**Reply by jeffpooley** [reply]
Thanks Gabe! totally understood. That display customization would be a good fix, but really no worry or rush….
